auditcare/management/commands/generate_request_report.py

```diff
@@ -0,0 +1,45 @@
+import csv
+import re
+
+from django.contrib.auth.models import User
+from django.core.management.base import LabelCommand
+
+from corehq.apps.users.models import WebUser
+from dimagi.utils.couch.database import iter_docs
+
+from auditcare.models import NavigationEventAudit
+
+
+def navigation_event_ids_by_user(user):
+    database = NavigationEventAudit.get_db()
+
+    return {row['id'] for row in database.view('auditcare/urlpath_by_user_date',
+        startkey=[user ],
+        endkey=[user, {}],
+        reduce=False,
+        include_docs=False,
+    )}
+
+def filter_non_domain_request(domain, request_path):
+    return request_path.startswith('/a/' + domain)
+
+def get_users(domain):
+    users = [u.username for u in WebUser.by_domain(domain)]
+    super_users = [u['username'] for u in User.objects.filter(is_superuser=True).values('username')]
```

Comment:
https://github.com/dimagi/auditcare/pull/24/commits/b4aa4d3e2e9cb31ba600c0d866450aa9aab340ea

I added the option, but what is the reasoning behind removing them? If it's going to be a transparent report, then it should be all access, not just ones from their users